media/js/mozorg/home/home-a.js

```diff
@@ -0,0 +1,174 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, Waypoint) {
+    'use strict';
+
+    var $navPage = $('#nav-page');
+
+    // waypoints
+    var introWaypoint;
+    var actionWaypoint;
+    var innovationWaypoint;
+    var firefoxWaypoint;
+
+    // intro slideshow
+    var $slideshow = $('#home-slideshow');
+    var mqIsTablet;
+
+    // encryption video
+    var encryptionVideoContainer = new Mozilla.VideoPosterHelper('#action');
+    var $video = $('#encryption-video');
+
+    // innovation section - aframe 3d demo
+    var $aframeIframe;
+    var $aframeShield; // sits over iframe to prevent scroll capture
+
+    // test for matchMedia
+    if ('matchMedia' in window) {
+        mqIsTablet = matchMedia('(min-width: 760px)');
+    }
+
+
+    // Intro slideshow
+    function startSlideshow() {
+        $slideshow.cycle({
+            fx: 'fade',
+            log: false,
+            slides: '> .slide',
+            speed: 1000,
+            startingSlide: 1, // start on group photo
+            timeout: 5000
+        });
+    }
+
+
+    // Encryption video
+    // use poster value from <video>
+    $('#encryption-video-container .moz-video-button').css({
+        'background-image': 'url(' + $video.attr('poster') + ')'
+    });
+
+    encryptionVideoContainer.init();
+
+    // track play and complete on video
+    $video.on('play', function() {
+        window.dataLayer.push({
+            'event': 'video-interaction',
+            'interaction': 'video - play',
+            'videoTitle': 'home-encryption'
+        });
+    }).on('ended', function() {
+        window.dataLayer.push({
+            'event': 'video-interaction',
+            'interaction': 'video - complete',
+            'videoTitle': 'home-encryption'
+        });
+    });
+
+    // Innovation section - aframe
+    function isCanvasSupported() {
+        var elem = document.createElement('canvas');
+        return !!(elem.getContext && elem.getContext('2d'));
+    }
+
+    if (isCanvasSupported()) {
```

Comment:
Looks like IE 11 can't handle it either. Will add some code to prevent IE < Edge from attempting the aframe.